Changelog.md

```diff
@@ -1,6 +1,10 @@
 ## Ignition Rendering
 
-### Ignition Rendering 3.X.X (2021-XX-XX)
+### Ignition Rendering 3.x.x (2021-XX-XX)
+
+1. CMake warning on Ogre versions that are not officially supported.
+   To disable the new warning is enough to enable the cmake option USE_UNOFFICAL_OGRE_VERSIONS
```

Comment:
`To disable the new warning, it is enough to ...`?

Reply:
done 2cdccf2